src/layer/types.js

```diff
@@ -12,10 +12,13 @@ import {TETHER_PLACEMENT} from './constants.js';
 /** LayersManager */
 export type LayersManagerPropsT = {
   children: React.Node,
+  // z-index to be set for all the layers.
+  zIndex?: ?number,
```

Comment:
if a `zIndex` property is provided, it needs to be non-null correct? same for the similar types below

```suggestion
  zIndex?: number,
```